edx_event_bus_redis/internal/consumer.py

```diff
@@ -93,18 +93,25 @@ class RedisEventConsumer(EventBusConsumer):
         consumer_name: unique name for consumer within a group.
         last_read_msg_id: Start reading msgs from a specific redis msg id.
         check_backlog: flag to process all messages that were not read by this consumer group.
+        claim_msgs_older_than: claim pending msgs from other consumers in the group with idle time older
+            than a specific time (in milliseconds).
+        has_pending_msgs: flag to process pending msgs first.
         db: Walrus object for redis connection.
         full_topic: topic prefixed with environment name.
         consumer: consumer instance.
     """
 
-    def __init__(self, topic, group_id, signal, consumer_name, last_read_msg_id=None, check_backlog=False):
+    def __init__(self, topic, group_id, signal, consumer_name, last_read_msg_id=None, check_backlog=False,
+                 claim_msgs_older_than=None):
```

Comment:
Is this flag documented anywhere? It seems like the default behavior will be to never claim, which seems wrong.

Reply:
(Not necessarily wrong that it won't claim messages, but that there's no doc explaining that you probably really do want this set to some other value. I think a sensible default would be somewhere in the range of a minute if we wanted to add one, since it needs to be long enough that an long-running jobs won't accidentally get double processed.

Reply:
@bmtcril It is documented above in line 96. Do you have a suggestion for a default idle time? This behavior is similar to kafka event bus by default as it also doesn't claim messages from other consumers.

Reply:
Hmmm I meant more in a "as a site operator deploying this, how would I know what this flag is and what to set it to", but it actually looks like we don't have any of that kind of documentation right now? I think it's unrelated to this PR, but would be good to provide a doc of how to run in production as well as a list of all of the settings and command line options in the docs.

Reply:
Ahh we commented at the same time. We could add some examples along with the tutor/devstack docs that we are working on in the other ticket explaining all the flags and their default values.